lib/routes/client-tokens/delete.js

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const db = require('../../db');
+
+module.exports = {
+  auth: {
+    strategy: 'authBearer'
```

Comment:
add a `scope: ['clients:write']` here. Or, bonus points for importing the constant.